composer.json

```diff
@@ -8,7 +8,7 @@
     ],
     "require": {
         "php" : ">=7.1",
-        "psr/container": "^2.0",
+        "psr/container": "^1.1.2 || ^2.0.2",
```

Comment:
do we need to lock in the patch versions?

Reply:
ah good call